### PR TITLE
Style/#103 trade view

### LINKED ISSUE #103: Improvement of TradeView

### PR DESCRIPTION
**Pull request by Enessetere**
closes #103

### CODE REVIEW

**Comment by Enessetere** [comment]
image

File: src/app/components/TradeModal.ts [hunk 1]
```diff
@@ -165,9 +189,90 @@ export class TradeModal extends EmptyModal {
...
+    return input;
+  }
+
+  private createButtons(
+    isBank: boolean,
+    animal: string,
+    count: number,
+  ): HTMLElement[] {
+    const buttons: HTMLElement[] = [];
+    buttons.push(this.createSingleButton(1, isBank, animal, count));
```
Comment (vieraboschkova) [inline src/app/components/TradeModal.ts:223]: Perhaps there is a way not to repeat next 4 lines?

**Review by vieraboschkova — CHANGES_REQUESTED** [review]
The boxes with numbers of offer and target are not aligned, please aligne them so they are one under another

**Comment by Enessetere** [comment]
image

**Review by DominikNowak — COMMENTED** [review]
Looks good to me. Could you add capitalize to the bank name?

**Review by DominikNowak — APPROVED** [review]
Well done

**Review by vieraboschkova — APPROVED** [review]
Very clean, good job

**Pull request merged**